### DISCUSSION TITLE
Integrate buildin FS prefab visualization

### DISCUSSION THREAD

**Discussion by StjerneIdioten** [StjerneIdioten/I3D-Blender-Addon#108] [Ideas & Feature Requests]
**Describe the solution you'd like**
A system where you can add premade components (i3d) from the fs $data directly in your blendfile to visualize the final model. 

**Describe alternatives you've considered**
Either importing the i3d and have it directly as a normal object or in a different scene. And maybe keep a reference to the i3d file so it can just be linked on export without converting it back to i3d, since that ressource is already available.

**Comment by kbrandwijk** [comment]
Would this also work for visualizing wheel configs? I think that would be a welcome addition, because currently you can only verify that configuration in-game.

**Reply by StjerneIdioten** [reply]
My thought was exactly with stuff like wheel configs in mind or other parts that aren't spawned in until you get in game (such as link arms).

The end result would be exporting imports of i3d files into the xml file.
